src/providers/ml_commons/agent_framework.ts

```diff
@@ -51,7 +51,7 @@ export class AgentFrameworkApiProvider implements ApiProvider {
       const response = (await openSearchClient.transport.request({
         method: 'POST',
         path: `/_plugins/_ml/agents/${agentId}/_execute`,
-        body: JSON.stringify({ parameters: { question: prompt, ...context?.vars } }),
+        body: JSON.stringify({ parameters: { input: prompt, ...context?.vars } }),
```

Comment:
instead of this, you can override the `buildInput` function in `SearchIndexRunner`
```typescript
  protected buildInput(spec: DSLSpec): {
    prompt: Parameters<ApiProvider['callApi']>[0];
    context: Parameters<ApiProvider['callApi']>[1];
  } {
    return {
      prompt: undefined,
      context: { vars: { input: spec.input } },
    };
  }
```